Terrain documentation (#1279)
Exawind/amr-wind: docs/sphinx/walkthrough/terrain.rst

> .. spelling:word-list::
> 
>    xy

Reviewer: ```suggestion
```

> .. spelling:word-list::
> 
>    xy

Reviewer: Can remove.
```suggestion
```

the values from a precursor simulation for both neutral and  non-neutral stratification. 
The terrain can be visualized by including ``io.int_outputs = terrain_blank`` in the input file. 

It is recommended to use the ``ProbeSampler`` to create the terrain-aligned output planes. The easiest method
to generate the text file for ``ProbeSampler`` is to write the STL as a text file and then use offsets in 
postprocessing to write the planes at different heights above the terrain. The terrain-aware output can 
also be used with FAST.Farm and FLORIS.

An example paraview visualization of the terrain is shown below (with three levels of refinement): 

.. image:: ./visualization/terrain_blank.png
    :width: 100%

Here is a sample content of precursor and inflow-outflow input files to drive terrain simulations: 

.. literalinclude:: ./terrain_precursor_inp.txt
   :linenos:

.. literalinclude:: ./terrain_inp.txt
   :linenos:

Setup using Python Tools
------------------------
The setup of the terrain files can be cumbersome to do by hand. A set of python tools are made available at 

> `literature <https://github.com/hgopalan/amrTerrain/tree/main/src/backend>`_. A more comprehensive set of tools 

Reviewer: ```suggestion
`this link <https://github.com/hgopalan/amrTerrain/tree/main/src/backend>`_. A more comprehensive set of tools 
```
